pkg/controller/useraccount/useraccount_controller.go

```diff
@@ -608,3 +644,59 @@ func newNSTemplateSet(userAcc *toolchainv1alpha1.UserAccount) *toolchainv1alpha1
 func ToIdentityName(userID string, identityProvider IdentityProvider) string {
 	return fmt.Sprintf("%s:%s", identityProvider.GetIdP(), userID)
 }
+
+func (r *ReconcileUserAccount) lookupAndDeleteCheUser(userAcc *toolchainv1alpha1.UserAccount) error {
+
+	// If the admin username is not set then Che user deletion is not configured, just return
+	if r.config.GetCheAdminUsername() == "" {
```

Comment:
do we have a username only, or a password/token as well?

Reply:
There is a password as well, but the main purpose of this line is just to have a way to disable this Che user deletion stuff if we want/need to. Instead of creating a new flag for that I figured just whether the username is configured or not in the member operator secret could serve as a gate for that purpose.

Reply:
ok, I see. Thanks for the explanation